### PR TITLE
Add Naval Strike levels + game mode

### PR DESCRIPTION
**Pull request by karllindmark**
@peter-budo:

Any other game modes / items you know of?

### CODE REVIEW

File: BF4Intel/src/main/java/com/ninetwozero/bf4intel/resources/maps/GameModeStringMap.java [hunk 1]
```diff
@@ -11,9 +11,11 @@
         put(8, "Squad Deathmatch");
         put(32, "Team Deathmatch");
         put(64, "Conquest Large");
+        put(64, "Conquest Large");
```
Comment (peter-budo) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/resources/maps/GameModeStringMap.java]: Conquest Large, Squad DM is there twice
Missing Capture the Flag, Air Superiority, Carrier Assault

Comment (karllindmark) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/resources/maps/GameModeStringMap.java]: I found a large JSON nap with like 30 game modes, so I didn't really know
which to pick LOL

**Comment by karllindmark** [comment]
@peter-budo: Fixed!

**Comment by peter-budo** [comment]
Merging, however I will try to find out which Squad DM we need to use. Can't do it from Mac, have to have valid browser plugin for BF to see who is playing on a server

**Comment by peter-budo** [comment]
Storing this for reference http://battlelog.battlefield.com/bf4/soldier/M1st3r_R3ap3r/battlereports/474438081/pc/ player has plenty of Squad DM

**Comment by karllindmark** [comment]
@peter-budo: Squad Deathmatch is already present in the map as far as I can tell (verified that it matches "gameMode: 8" from the website as well).

**Comment by peter-budo** [comment]
@karllindmark point is that there 2 squad dm in that class and one should go...

**Comment by karllindmark** [comment]
@peter-budo 

Seems like "Squad Rush" may be more appropriate as the value for "4" (according to the map below), however not sure which of these are used in BF4 - all of them? Only a fraction?

```
{
  "TEAMDEATHMATCH": 32,
  "RUSHSMALL": 2,
  "CONQUESTASSAULTLARGE": 128,
  "UNKNOWN": 1,
  "CARRIERASSAULTSMALL": 134217728,
  "GUNMASTER": 512,
  "DOMINATION": 1024,
  "SQUADDEATHMATCH": 8,
  "CARRIERASSAULT": 33554432,
  "OBJECTIVERAID": 262144,
  "CONQUESTSMALL": 1,
  "BOMBSQUAD": 1048576,
  "TEAMDEATHMATCH_FIRETEAM": 4096,
  "NONE": 1,
  "COMBATMISSION": 8192,
  "CONQUESTASSAULTSMALL": 256,
  "AIRSUPERIORITY": 8388608,
  "TANKSUPERIORITY": 131072,
  "CONQUESTLARGE": 64,
  "SQRUSH": 4,
  "SCAVENGER": 4194304,
  "SPORT": 65536,
  "SQDM": 8,
  "RUSHLARGE": 2,
  "CONQUEST": 1,
  "RUSH": 2,
  "OBLITERATION": 2097152,
  "FIRETEAM_SURVIVOR": 32768,
  "TEAMDEATHMATCHC": 2048,
  "ONSLAUGHT": 16,
  "SECTORCONTROL": 16384,
  "CARRIERASSAULTLARGE": 67108864,
  "SQUADRUSH": 4,
  "CAPTURETHEFLAG": 524288,
  "ELIMINATION": 16777216
}
```

**Comment by peter-budo** [comment]
@karllindmark 
Only fraction of these are in BF4. Squad rush was there at beginning but I guess they scrap it as not many people play it. To check all available modes you can always check it in Server Browser when you click down arrow on black strip bellow Recommended Servers. Here you can customize what game mode, dlc, and other settings you want to apply to your default server selection to show

**Pull request merged**